src/emulator/host/src/app.cpp

```diff
@@ -132,8 +142,23 @@ bool install_vpk(Ptr<const void> &entry_point, HostState &host, const std::wstri
     title_base_path += host.io.title_id;
     bool created = fs::create_directory(title_base_path);
     if (!created) {
-        LOG_INFO("{} already installed, launching application...", host.io.title_id);
-        return true;
+        uint8_t status = 0;
+        while (handle_events(host) && (status == 0)) {
+            ImGui_ImplSdlGL2_NewFrame(host.window.get());
+            glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
+            DrawUI(host);
+            DrawReinstallDialog(host, &status);
+            glViewport(0, 0, static_cast<int>(ImGui::GetIO().DisplaySize.x), static_cast<int>(ImGui::GetIO().DisplaySize.y));
+            ImGui::Render();
+            ImGui_ImplSdlGL2_RenderDrawData(ImGui::GetDrawData());
+            SDL_GL_SwapWindow(host.window.get());
+        }
+        if (status == 1) {
```

Comment:
Yes is 2 and No is 1 above. Here you're checking for 1 and 0?
Also these should be enums, not arbitrary magic values.